scripts/create-update-issues.sh

```diff
@@ -0,0 +1,240 @@
+#!/bin/bash
+
+set -euo pipefail
+
+DEFAULT_REF="HEAD"
+DEFAULT_LABEL="client-controller-update"
+
+print-usage() {
+  cat <<EOT
+Detects major-bumped packages in a release commit and creates tickets in clients
+that instructs engineers to upgrade to them.
+
+Usage: $0 [--ref REF] [--no-dry-run]
+
+OPTIONS:
+
+--ref REF, -r REF       The release commit to inspect.
+--no-dry-run            By default, this script won't do anything. Pass this
+                        option to override that.
+--help, -h              You're looking at it ;)
+EOT
+}
+
+existing-issue-found() {
+  local repo="$1"
+  local package_name="$2"
+  local version="$3"
+
+  local all_issues
+  if ! all_issues=$(gh issue list --repo "$repo" --label "$DEFAULT_LABEL" --state open --json number,title,url 2>&1); then
```

Comment:
Hmm. This seems a bit inefficient, fetching this list once per published package.

Maybe we could fetch the list of existing issues just twice (for extension and mobile respective) and pass it into this function instead. Something like this: https://github.com/MetaMask/core/pull/6143